shared/logutil/logutil.go

```diff
@@ -0,0 +1,83 @@
+package logutil
```

Comment:
Please add a package level comment for godocs.

Reply:
Done